server/main.js

```diff
@@ -2,17 +2,20 @@ import { Meteor } from 'meteor/meteor';
 import '../imports/startup/server/fixtures.js';
 
 // Methods
-import '../imports/api/login.js';
 import '../imports/api/users/methods.js';
 import '../imports/api/projects/methods.js';
+import '../imports/api/requests/methods.js';
 
 // Collections
 import '../imports/api/projects/projects.js';
+import '../imports/api/requests/requests.js';
 
 // Publications
 import '../imports/api/projects/server/publications.js';
 import '../imports/api/users/server/publications.js';
+import '../imports/api/requests/server/publications.js';
 
 Meteor.startup(() => {
   // code to run on server at startup
+  process.env.MAIL_URL = 'smtp://meteor.no.reply%40gmail.com:Password&123@smtp.gmail.com:465';
```

Comment:
The MAIL_URL should not be hardcoded.  You could leave it as an environment variable, but then check to see if it is set before doing anything with mail.  Or you could use Meteor Settings.  See https://guide.meteor.com/deployment.html#environment

This might even be a setting that is saved in the database, settable from the Admin Dashboard.  But then again, this is something that probably doesn't change often or at all, so the above options might be more suitable.